src/cv2pdb.h

```diff
@@ -236,8 +244,11 @@ class CV2PDB : public LastError
 	int cbDwarfTypes;
 	int allocDwarfTypes;
 
-	int nextUserType;
-	int nextDwarfType;
+	static constexpr int BASE_USER_TYPE = 0x1000;
+	static constexpr int BASE_DWARF_TYPE = 0x1000;

```

Comment:
I don't think we need two different constants here, as the value refers to the beginning of user types in the target format (CodeView).